resources/sass/app.scss

```diff
@@ -17,6 +17,9 @@
 $form-control-border-color: #b6bfc6;
 $form-control-border-radius: 2px;
 $focus-border-opacity: .5;
+$ellipsis-menu-btn-color: #ffffff;
```

Comment:
@mcraeteisha this PR looks great, I love how ellipsis menu looks! I only have a little observation that we confirmed with @hvanlear 
The default background color for the button is white, when you hover the rows, the background of the row is gray and the button still white, and should maintain the "flat" aspect (I mean should have the save gray color). I think probably you can try to change this variable color from `#ffffff`to transparent and see if that fix the issue.
Great Job!